lumberjack.go

```diff
@@ -205,6 +211,13 @@ func (l *Logger) openNew() error {
 			return fmt.Errorf("can't rename log file: %s", err)
 		}
 
+		// Compress the backup if enabled
+		if l.CompressBackups == true {
```

Comment:
this should really just be `if l.CompressBackups { ` - no need to check == true.